src/test/unit/test_spans_container.py

```diff
@@ -82,6 +83,22 @@ def test_spans_container_end_function_send_only_on_errors_mode_false_not_effecti
     assert reported_ttl is not None
 
 
+def test_spans_container_end_function_with_error_double_size_limit(monkeypatch, dummy_span):
+    long_string = "v" * Configuration.get_max_entry_size() * 3
+    monkeypatch.setenv("LONG_STRING", long_string)
+    event = {"k": long_string}
+    SpansContainer.create_span(event)
+    SpansContainer.get_span().start()
+    start_span = copy.deepcopy(SpansContainer.get_span().function_span)
+    SpansContainer.get_span().add_exception_event(Exception("Some Error"), inspect.trace())
+
+    SpansContainer.get_span().end(event)
+
+    end_span = SpansContainer.get_span().function_span
+    assert len(end_span["event"]) > len(start_span["event"]) * 1.8 + 1
```

Comment:
why do you compare to the start span?
You built this string in the test itself - use `long_string` explicitly,
and then you can drop this weird formula `*1.8+1`

Reply:
I need to compare to the start span because I want to make sure that we send extra data in case of an error.
For example, someone will change the size_limit to 5000 and in case of an error we won't double the size_limit. In this case I want the test to fail